connector/src/main/java/org/geysermc/connector/scoreboard/ScoreboardUpdater.java

```diff
@@ -133,11 +142,11 @@ private long getTimeTillNextAction() {
 
     @RequiredArgsConstructor
     @Getter
-    public static class ScoreboardSession {
+    public static final class ScoreboardSession {
         private final GeyserSession session;
         private final AtomicInteger pendingPacketsPerSecond = new AtomicInteger(0);
         private int packetsPerSecond;
-        private int millisBetweenUpdates = FIRST_MILLIS_BETWEEN_UPDATES;
+        private boolean justSwitchedToSelf;
```

Comment:
Can you add a Javadoc comment here explaining what this is for?

Reply:
I tried to explain the purpose of it on line 87